eglibc-2.9: allow building with GNU make >= 3.82
    
GNU make >= 3.82 does not allow mixed implicit and normal rules

Because eglibc-2.9 is superseded by newer versions it makes no sense
to submit this patch upstream.

Upstream-Status: Inappropriate [other]

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>

diff -Nurd libc.orig//manual/Makefile libc/manual/Makefile
--- libc.orig//manual/Makefile	2011-09-13 14:49:57.428424088 +0200
+++ libc/manual/Makefile	2011-09-13 14:50:47.009620484 +0200
@@ -242,7 +242,11 @@
 .PHONY: stubs
 stubs: $(objpfx)stubs
 endif
-$(objpfx)stubs ../po/manual.pot $(objpfx)stamp%:
+$(objpfx)stubs ../po/manual.pot:
+	$(make-target-directory)
+	touch $@
+
+$(objpfx)stamp%:
 	$(make-target-directory)
 	touch $@
 
